### ISSUE TITLE
Android: Widget with ImageView not showing up

### ISSUE THREAD

**Issue by m1ga** [AndreaVitale/av.imageview#26]
Might be related to https://github.com/AndreaVitale/imageview/issues/16
I have a basic widget:
```
<Alloy>
	<View id="view_all">
		<ImageView id="image" module="av.imageview"/>
		<Label id="title"/>
	</View>
</Alloy>
```

```
"#view_all": {
	width:Ti.UI.FILL,
	height: Ti.UI.FILL,
	left:0,
	borderWidth: 1,
	borderColor: "#fff"
}

"#image":{
	left:0,
	height:80,
	width: Ti.UI.FILL,
	touchEnabled: false,
	contentMode: Alloy.Globals.CONTENT_MODE_FILL
}
"#title":{
	left: 10,
	top:10,
	color:"#fff"
}

```
and I create and include it into a TableView. On iOS the image shows up on Android it stays blank

**Comment by m1ga** [comment]
Just a short update because I did some debugging today:

```xml
<TableView id="tbl">
	<TableViewRow>
		<Widget src="img" image="http://placehold.it/100x100" width="100" height="100"/>
	</TableViewRow>
	<!-- working because it uses a processProperties:
	<TableViewRow>
		<ImageView module="av.imageview" image="http://placehold.it/300x100" width="100" height="100"/>
	</TableViewRow>
	 -->
</TableView>
```

Added some debug output in different methods:

widget in tableview with image as setImage
```
[INFO]  __SetImage: (main) [266,266] Set : http://placehold.it/100x100->true
[INFO]  __1: (main) [0,266] com.miga.testav.Test_avActivity@8d35d8
[INFO]  __CreateView: (main) [1,267] create view av.imageview.ImageViewProxy@664e3a2
[INFO]  ___: (main) [8,275] set layout
[INFO]  __1: (main) [257,257] org.appcelerator.titanium.TiActivity@e962155
[INFO]  __CreateView: (main) [0,257] create view av.imageview.ImageViewProxy@664e3a2
[INFO]  ___: (main) [2,259] set layout
```

module in tableview with image as parameter
```
INFO]  handleCreationDict: (main) [252,252] Handle dict: http://placehold.it/300x100
[INFO]  __SetImage: (main) [0,252] Set : http://placehold.it/300x100->true
[INFO]  __1: (main) [0,252] com.miga.testav.Test_avActivity@8d35d8
[INFO]  __CreateView: (main) [2,254] create view av.imageview.ImageViewProxy@b4d2b6d
[INFO]  ___: (main) [9,263] set layout
[INFO]  processProperties: (main) [6,269] Process prop: http://placehold.it/300x100
[INFO]  __1: (main) [195,195] org.appcelerator.titanium.TiActivity@ab8d40c
[INFO]  __CreateView: (main) [0,195] create view av.imageview.ImageViewProxy@b4d2b6d
[INFO]  ___: (main) [1,196] set layout
[INFO]  processProperties: (main) [1,197] Process prop: http://placehold.it/300x100
[INFO]  ___: (main) [179,376] http://placehold.it/300x100 is ready
```

When using the module with the image as a property it works (even inside a widget) because it will run processProperties twice because createView is called twice.
If you use the `setImage()` method to set a dynamic image it will run `setImage` once and the creates the view twice and doesn't run  processProperties/setImage at the end again -> image is empty.

Still playing around with it to see how to fix that behaviour

**Comment by m1ga** [comment]
Looks like it was easier than I thought: https://github.com/AndreaVitale/imageview/pull/27
I don't have a big project to test it but my demo tableview example is fixed. Perhaps someone who is using it in a more complex scenario can check the binary in the PR!

**Comment by m1ga** [comment]
Not fixed by your refactoring. Here again the log:

> [INFO]  AVImageViewProxy: (main) [324,324] Proxy setImage http://placehold.it/100x100
> [INFO]  AVImageView: (main) [22,346] set layout
> [INFO]  AVImageViewProxy: (main) [5,351] create imageView
> [INFO]  AVImageView: (main) [3,354] processProperties
> [INFO]  AVImageView: (main) [5,359] AVImage set image http://placehold.it/100x100
> [INFO]  AVImageView: (main) [278,278] set layout
> [INFO]  AVImageViewProxy: (main) [0,278] create imageView
> [INFO]  AVImageView: (main) [1,279] processProperties

Test project: http://migaweb.de/test_av.zip
The module inside is yours with the log output.

Again the problem is, that the `createView` is called twice but the AVImageViewProxy `setImage()` is only called at the beginning. After the second `createView` there is nothing inside the properties and processProperties can't set the image.

BTW: I don't know why it is called twice :)

**Comment by AndreaVitale** [comment]
I published a new Android version [2.1.3](https://github.com/AndreaVitale/imageview/releases/tag/Android-2.1.3) that seems to fix your issue. (tested with your attached project). Could you please verify if it is also fixed for you? Thanks.

**Comment by m1ga** [comment]
Looking good! Will do a test on a different project tomorrow but at the moment its looking very nice
